### DISCUSSION TITLE
mstsc freezes the rdp connection

### DISCUSSION THREAD

**Discussion by skumaranece** [FreeRDP/FreeRDP#7252] [Q&A] [answered]
In ubuntu 18.04 side:

Am running the freerdp server.

:~$ freerdp-shadow-cli -version
[11:12:13:322] [2747:2747] [INFO][com.freerdp.server.shadow] - FreeRDP version 2.2.0 (git n/a)

:~$freerdp-shadow-cli -auth

----------------------------------------------------------------------------------------------------------------------------------------------

In windows 10 side: running the mstsc client

After entering the user name and password, am able to get the linux machine in remote desktop.
But am not able to do provide any input through keyboard or mouse.

Almost the remote desktop screen looks like hang.

**Comment by akallabeth** [comment]
shadow server is currently limited (e.g. tech demo, beta) and you´re using an older version with known bugs.

**Comment by skumaranece** [comment]
@akallabeth Thanks for the info. Do we have the working version of freerdp server for mstsc ?

**Reply by akallabeth** [reply]
yes, shadow works with `mstsc`.
but you need to be vigilant, there are limitations what the server can do. (e.g. you need to connect with the exact same resolution of the remote or you will not see anything)

**Comment by skumaranece** [comment, marked as answer]
Follow the below steps to get the remote desktop through mstsc:

1. press windows + r button
2. in run, open the mmc
3. file -> gpedit.mds
4. Use the branch: Computer Configuration->Administrative Templates->Windows Components->Remote Desktop Services->Remote Desktop Connection client->do not allow hardware acceleration decoding, set to enable
(refer: https://admx.help/?Category=Windows_10_2016&Policy=Microsoft.Policies.TerminalServer::TS_CLIENT_DISABLE_HARDWARE_MODE)
5. in mstsc display tab --> choose the color depth of the remote session(32bit) - try other bit combination which suits for your device
